code/game/objects/structures/tables_racks.dm

```diff
@@ -615,6 +615,18 @@
 		return
 	return ..()
 
+/obj/structure/table/MouseDrop_T(atom/movable/O, mob/user)
+	if(O != user || !ishuman(usr) || !in_range(src, usr) || usr.incapacitated() || usr.lying) // Doesn't work if you're not dragging yourself, not a human, not in range or incapacitated
```

Comment:
`!in_range(src, usr)` should be `!Adjacent(user)`

replace usr with user for these, usr isn't very sane